### PR TITLE
Update Drupal and contrib modules.

### PR DESCRIPTION
**Pull request by HelenaEksler**
#112

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@amitaibu PR is ready.

File: shoov/drupal-org.make [hunk 1]
```diff
@@ -113,14 +113,14 @@ projects[views][subdir] = "contrib"
...
-projects[views_bulk_operations][version] = "3.2"
+projects[views_bulk_operations][version] = "3.3"
 
 ; Development
 projects[devel][subdir] = "development"
 projects[devel][version] = "1.5"
 
 projects[migrate][subdir] = "development"
-projects[migrate][version] = 2.5
+projects[migrate][version] = 2.8
```
Comment (amitaibu) [inline shoov/drupal-org.make]: lets keep migrate on 2.5, otherwise migrate throws errors

Comment (amitaibu) [inline shoov/drupal-org.make]: actually leave on 2.8 -- maybe it was just on my local.

Comment (HelenaEksler) [inline shoov/drupal-org.make]: I installed on my local. I see an error `Passing an empty first parameter to a migration constructor is now deprecated - pass through the arguments array passed to the leaf class instead.` but all data is migrated. So I think I'll change the version back for now.

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**